Sources/FTPropertyWrappers/GenericPassword.swift

```diff
@@ -3,27 +3,38 @@ import Foundation
 @propertyWrapper
 open class GenericPassword<T: Codable>: KeychainItemPropertyWrapper<T> {
 
-    @QueryElement(key: kSecAttrAccount) open var account: String?
-    @QueryElement(key: kSecAttrService) open var service: String?
+    @QueryElement(key: kSecAttrAccount) open private(set) var account: String?
+    @QueryElement(key: kSecAttrService) open private(set) var service: String?
     @QueryElement(key: kSecAttrAccessControl,
                   constraints: [.override(kSecAttrAccessible)]) open private(set) var accessControl: SecAccessControl?
 
     override open var itemClass: CFString { kSecClassGenericPassword }
 
-    override open var primaryKey: Set<String> { [kSecAttrService as String] }
+    override open var primaryKey: Set<CFString> { [
+                                                    kSecAttrAccount,
+                                                    kSecAttrService,
+                                                ] }
 
     override open var wrappedValue: T? {
         get { super.wrappedValue }
         set { super.wrappedValue = newValue }
     }
 
-    public init(serviceIdentifier: String, refreshPolicy: KeychainDataRefreshPolicy = .onAccess, defaultValue: T? = nil) {
+    public init(service: String,
```

Comment:
Just to let you know: You are using Objective-C indentation, look at the standard library (first function I found):

https://github.com/apple/swift/blob/master/stdlib/public/core/Array.swift#L1137

Reply:
Would you provide me please some clearer example of desired indentation? I do use default indentation as suggested by Xcode. 

It may be similar to Objc indentation which is similar, but to my knowledge, Objc aligns lines to ":" sign, not to the first letter of label.

Reply:
OK, you are using similar indentation to Objective-C. 🙂 

Other examples:

https://github.com/apple/swift/blob/master/stdlib/public/core/CollectionDifference.swift#L298
https://github.com/apple/swift/blob/master/stdlib/public/core/Dictionary.swift#L474
https://github.com/apple/swift/blob/master/stdlib/public/core/Dictionary.swift#L526